libs/ui/src/theme/global/mediaQueries.ts

```diff
@@ -0,0 +1,19 @@
+export const widthBreakpoint = {
+  mobileSm: 350,
+  mobileMd: 378,
```

Comment:
Will we really have css changes between small and medium mobile? Seems like one could be eliminated but I am fine either way

Reply:
I think that's a good point and in actuality it's probably unlikely, but may not hurt to have the extra breakpoints in there incase it does become a distinction for a feature later on